steem/utils.py

```diff
@@ -354,3 +362,78 @@ def strfdelta(tdelta, fmt):
 
 def is_valid_account_name(name):
     return re.match('^[a-z][a-z0-9\-.]{2,15}$', name)
+
+
+def compat_compose_dictionary(dictionary, **kwargs):
+    """
+    This method allows us the one line dictionary composition that is offered by the ** dictionary unpacking
+    available in 3.6.
+
+    :param dictionary: the dictionary to add the kwargs elements to.
+    :param kwargs: a set of key/value pairs to add to `dictionary`.
+    :return: the composed dictionary.
+    """
+    composed_dict = dictionary.copy()
+    for pairs in kwargs.items():
+        composed_dict[pairs[0]] = pairs[1]
```

Comment:
Just for simplicity I'd replace the for loop with `composed_dict.update(kwargs)`

Reply:
duh. Thanks!